src/alto-ui/Dropdown/components/DropdownItem/DropdownItem.scss

```diff
@@ -13,6 +13,7 @@
   display: flex;
   align-items: center;
   color: $coolgrey-60;
+  fill: $coolgrey-60;
```

Comment:
why ? normally Icons use font color

Reply:
It's change for circle in radio button icon. Font color doesn't help

Reply:
But I can also pass `ownProps` to circle in Radio component

Reply:
Improved